src/Raven.Server/Config/Categories/IndexingConfiguration.cs

```diff
@@ -493,6 +493,13 @@ public virtual bool RunInMemory
         [DefaultValue(false)]
         [IndexUpdateType(IndexUpdateType.Refresh)]
         public bool CoraxIncludeSpatialDistance { get; set; }
+        
+        [Description("The maximum amount of memory that Corax can use for a memoization clause during query processing")]
+        [DefaultValue(128)]
+        [SizeUnit(SizeUnit.Megabytes)]
+        [IndexUpdateType(IndexUpdateType.None)]
```

Comment:
Shouldn't this be Refresh and handled in the Update method?

Reply:
done in https://github.com/ravendb/ravendb/pull/17088/commits/cd05998cd6c2382d0835d0d73f7241f224d1d238